internal/cmd/configssh.go

```diff
@@ -165,6 +166,52 @@ func configSSH(configpath *string, remove *bool, next *bool) func(cmd *cobra.Com
 	}
 }
 
+// binPath returns the path to the coder binary suitable for use in ssh
+// ProxyCommand.
+func binPath() (string, error) {
+	exePath, err := os.Executable()
+	if err != nil {
+		return "", xerrors.Errorf("get executable path: %w", err)
+	}
+
+	// On Windows, the coder-cli executable must be in $PATH for both Msys2/Git
+	// Bash and OpenSSH for Windows (used by Powershell and VS Code) to function
+	// correctly. Check if the current executable is in $PATH, and warn the user
+	// if it isn't.
+	if runtime.GOOS == "windows" {
+		binName := filepath.Base(exePath)
+		// We use safeexec instead of os/exec because os/exec returns paths in
+		// the current working directory, which we will run into very often when
+		// looking for our own path.
+		pathPath, err := safeexec.LookPath(binName)
```

Comment:
safeexec works on my windows machine in a test program I wrote, but never seems to work in the coder-cli cross-compiled builds from my workspace. I wonder if it's caused by cross-compile or if there's some BS at play